ChangeLog.md

```diff
@@ -1,4 +1,11 @@
-﻿## 2017.01.18 - Version 3.4.0
+﻿## 2017.02.20 - Version 3.5.0
```

Comment:
Please do not edit this.  Instead, please edit the changelog in your directory - the changes should go under the 'Current Release' section